.github/workflows/build_and_test.yml

```diff
@@ -1,6 +1,9 @@
 name: Build and Test
 
-# See https://docs.github.com/en/actions/reference/events-that-trigger-workflows
+concurrency:
```

Comment:
this was recommended by go/github-actions. basically if i spam multiple commits it makes sure to only run the job on the most recent commit, saving us minutes